### PR TITLE
Feature/loading animations

### PR DESCRIPTION
**Pull request by ror-shubham**
Added loading animations. Removed depreciated 'vuex-loading' and added 'vue-wait'.

### CODE REVIEW

File: src/app.scss [hunk 1]
```diff
@@ -64,6 +64,9 @@ html, body {
   display:flex;
   justify-content:center;
 }
+.loader {
+  margin: 40%;
+}
```
Comment (davidyuk) [inline src/app.scss]: I think it is better to create a wrapper around AeLoader than add stuff to global styles.

File: src/screens/generationList/generationList.vue [hunk 2]
```diff
@@ -16,65 +16,70 @@
...
-                <relative-time :ts="currentTime - b.keyBlock.time"/>
-              </span>
-            </td>
-          </template>
-          <template v-else>
-            <td colspan="3">Loading..</td>
+          <template>
+            <v-wait for="loading generation list">
+              <td slot="waiting">
+                <ae-loader slot="waiting"/>
```
Comment (davidyuk) [inline src/screens/generationList/generationList.vue]: The second `slot="waiting"` looks useless.

File: src/screens/generationList/generationList.vue [hunk 3]
```diff
@@ -16,65 +16,70 @@
...
-            <td>
-              <span class="field-name">time</span>
-              <span class="number">
-                <relative-time :ts="currentTime - b.keyBlock.time"/>
-              </span>
-            </td>
-          </template>
-          <template v-else>
-            <td colspan="3">Loading..</td>
+          <template>
```
Comment (davidyuk) [inline src/screens/generationList/generationList.vue]: This `<template >` has no sense.

**Comment by ror-shubham** [comment]
@davidyuk Please have a look again on recent commits,

File: src/screens/status/status.vue [hunk 4]
```diff
@@ -40,10 +40,10 @@
         <h3>top</h3>
         <pre>{{ nodeStatus.top }}</pre>
       </template>
-      <ae-loader v-else class="loader"/>
+      <loader v-else class="loader"/>
```
Comment (davidyuk) [inline src/screens/status/status.vue]: `class="loader"` looks unnecesary.

File: src/partials/recentBlocks/recentBlocks.vue [hunk 5]
```diff
@@ -39,22 +39,24 @@
           </template>
         </tr>
       </table>
+      <div v-else>
```
Comment (davidyuk) [inline src/partials/recentBlocks/recentBlocks.vue]: Here and below: is wrapper necessary? Can't it be just `<loader v-else />`?

File: package.json [hunk 6]
```diff
@@ -18,7 +18,7 @@
     "vue-popperjs": "^1.2.6",
     "vue-router": "^3.0.1",
     "vuex": "^3.0.1",
-    "vuex-loading": "^0.2.2"
+    "vue-wait": "^1.3.2"
```
Comment (davidyuk) [inline package.json]: So, this package is unused, let's remove it.

**Pull request merged**